perl6/lib/Cutie.pm

```diff
@@ -1,3 +1,3 @@
 role Cutie {
-  has Bool $.accept-hugs is rw = False;
+  method accept-hugs { … }
```

Comment:
omg really? You need to use the ellipsis character in Perl6?! Or it at least accepts it?